tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -502,6 +502,14 @@ function () {
 			}
 		);
 
+		add_filter(
+			'webp_uploads_image_sizes_with_additional_mime_type_support',
+			function( $sizes ) {
+				$sizes['1536x1536'] = '1536x1536';
+				return $sizes;
+			}
+		);
```

Comment:
Let's remove this, as this size does not have to exist for each image - that doesn't belong here. We shouldn't include checks for specific sizes here. Let's also remove the `1536x1536`-related assertion below.

Reply:
Yep, removed.